src/crewai/llm.py

```diff
@@ -984,6 +984,27 @@ def call(
                 # whether to summarize the content or abort based on the respect_context_window flag
                 raise
             except Exception as e:
+                unsupported_stop = "Unsupported parameter" in str(e) and "'stop'" in str(e)
+
+                if unsupported_stop:
+                    if "additional_drop_params" in self.additional_params and isinstance(self.additional_params["additional_drop_params"], list):
+                        self.additional_params["additional_drop_params"].append("stop")
+                    else:
+                        self.additional_params = {"additional_drop_params": ["stop"]}
```

Comment:
QQ: Why don't we add `stop` by default in `additional_drop_params` if we'll retry it with it anyways?